app/services/course_search_service.rb

```diff
@@ -14,9 +14,21 @@ def call(**args)
   def call
     scope = course_scope.findable
 
-    scope = scope.ascending_canonical_order if sort_by_provider_ascending?
-    scope = scope.descending_canonical_order if sort_by_provider_descending?
-    scope = scope.by_distance(origin: origin) if sort_by_distance?
+    if sort_by_provider_ascending?
+      scope = scope.ascending_canonical_order
+      scope = scope.select("provider.provider_name", "name")
+    end
```

Comment:
Can the results be sorted by more than one field at a time? If not, which I think is the case, is it worth being clearer about this in the code:

```ruby
if sort_by_provider_ascending?
  ...
elsif sort_by_provider_descending?
  ...
elsif sort_by_distance?
  ...
end
```

Reply:
done